.github/workflows/release.yml

```diff
@@ -0,0 +1,95 @@
+name: Release 
+
+on:
+  push:
+    branches: 
+      - main
+      - prerelease/major
+      - prerelease/minor
+
+  workflow_dispatch:
+    inputs:
+      version:
+        default: 'patch'
+        description:
+          'The version override: patch, minor or major'
+        required: false
+
+jobs:
+  release:
+    # Only run if:
+    # - The commit message does not contain `[skip release]`
+    # - OR the workflow was manually triggered and has a `version` string
+    if: ${{!contains(github.event.head_commit.message, '[skip release]')}}
+    runs-on: ubuntu-latest
+
+    steps:
+      ## First, we'll checkout the repository. We don't persist credentials because we need a
+      ## Personal Access Token to push on a branch that is protected. See
+      ## https://github.com/cycjimmy/semantic-release-action#basic-usage
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+          fetch-depth: 0 # Used for conventional commit ranges
+
+      - uses: Workday/canvas-kit-actions/install@v1
+        with:
+          node_version: 18.x
```

Comment:
I assume our other actions sue this node version, can this be updated at all to 22 or does that seem scary?

Reply:
we could try to bump to 22, i don't think it will impact hardly